src/test/components/__snapshots__/GlobalTrack.test.js.snap

```diff
@@ -17,6 +17,11 @@ process: \\"tab\\" (222)"
         type="button"
       >
         Content Process
+        <div
+          class="timelineTrackNameButtonAdditionalDetails"
+        >
+          222
+        </div>
```

Comment:
you probably need to regenerate these snapshots with the new change, and commit the new version of the snapshot

Reply:
DONE